### PR TITLE
Completed Cart and Auth pages

### PR DESCRIPTION
**Pull request by Pratham82**
- [x] Completed cart page
- [x] Completed Login page
- [x] Completed sign up page

UI:
Login page (click on user icon):
image

Sign up page (click on user icon):
image

Cart:
image

### CODE REVIEW

**Comment by ashifkhn** [comment]
Looks nice. Good to go.

**Review by pnchinmay — APPROVED** [review]
Looks good :100:

File: src/pages/auth/login.html [hunk 1]
```diff
@@ -47,5 +47,119 @@ <h6 class="h6">Shop All</h6>
...
+              <button class="btn primary-filled-btn">Sign Up</button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <script>
+      let content = document.getElementsByClassName('content')
+      let tabLinks = document.getElementsByClassName('tabLinks')
+
```
Comment (arulk8) [inline src/pages/auth/login.html]: move this to a separate js file

**Review by arulk8 — APPROVED** [review]

File: src/pages/auth/login.html (hunk 1 above)
Comment (arulk8) [reply src/pages/auth/login.html]: looks good to me

Comment (Pratham82) [reply src/pages/auth/login.html]: Thanks for the review, I'll fix this

**Pull request merged**